default-templates/server/index.html

```diff
@@ -27,6 +27,11 @@ <h1>Welcome to the Solid Prototype</h1>
     If you have not already done so, please create an account.
   </p>
 
+  <p class="lead hidden" id="loggedIn">
+    You are logged in as
+    <a href="#" id="profileLink"></a>.
```

Comment:
This does not appear to show the user their logged in WebID, nor any label associated with that WebID.  I submit that both of these must be available -- not least because I may have the same label on multiple WebIDs, but only want to associate one WebID with some activity or data.